extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -332,14 +332,7 @@
 
     <profiles>
         <profile>
-            <id>genpolicy</id>
-            <activation>
-                <!-- activated when Ranger version is identical to required-->
-                <property>
-                    <name>ranger.version</name>
-                    <value>2.4.0</value>
-                </property>
```

Comment:
Here removes the property-based activation introduced previously in #4585, since it's not activated with `ranger. version` set in project parent pom in value `2.4.0` (which requires more investigation in profile mechanism).

cc @Yikf

Reply:
Thanks for ping me, So do we have to specify the profile manually now?

Reply:
Yes, and the profile name changed to `gen-policy`.